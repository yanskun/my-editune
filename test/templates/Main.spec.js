import { createLocalVue, shallowMount } from "@vue/test-utils"
import ElementUI from "element-ui"
import Component from "~/components/templates/Top"

const localVue = createLocalVue()
localVue.use(ElementUI)

describe("Top", () => {
  test("snapshot", () => {
    const wrapper = shallowMount(Component, {
      propsData: {},
      localVue
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
