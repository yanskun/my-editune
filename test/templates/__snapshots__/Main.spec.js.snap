// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Top snapshot 1`] = `
<div>
  <app-header-stub />
  <el-container-stub />
</div>
`;
